### DISCUSSION TITLE
Sequence length based alignment algorhythm selection

### DISCUSSION THREAD

**Discussion by balazshorvathreal** [bbuchfink/diamond#448] [Ideas]
Dear Benjamin, 
do you plan implementing length based alignment algorhythm selection for the Diamond? I mean an additional option for long or short algo usage based on input read/sequence length. E.g. above 400-500nt length Diamond uses the --long-reads feature and below this (or parametrized) threshold chooses the default short read algo. 
It will be useful for creating one combined (instead of two - contig & remained read level) daa file that contains alignment of de novo assembled contigs and non remappable reads with low abundance. I suppose this kind of results should be more consistent and contain less false positive homologies with preserved sensitivity. Of course Megan need some updates to process this kind of hybrid datasets. 
Bests: Balázs

**Comment by bbuchfink** [comment]
I think the best solution for this would be to allow merging of DAA files, so you can simply combine results from different runs. It's been on my todo list and will hopefully be implemented soon.

**Reply by balazshorvathreal** [reply]
Thank you, it sounds marvellous! 

Cheers: Balázs
